### PR TITLE
refactor(*): Move config server integration to kork-config

### PR DESCRIPTION
**Pull request by scottfrederick**
Following https://github.com/spinnaker/kork/pull/332, remove config server setup and services that was moved from `clouddriver` to `kork-config`.

### CODE REVIEW

**Comment by scottfrederick** [comment]
@ezimanyi @ethanfrogers @cfieber This is ready for review now that the `kork-config` PR has been merged and kork has been autobumped in clouddriver.

**Comment by scottfrederick** [comment]
@ezimanyi How many times did I compare those properties? Not enough, apparently. I compared all the services and halyard, but this looks like the set from halyard that I didn't normalize properly. I'll make the fix in `kork-config` and wait for another bump to test this again.

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy [hunk 1]
```diff
@@ -59,26 +61,7 @@ import java.security.Security
 ])
 @EnableScheduling
 class Main extends SpringBootServletInitializer {
-
-  private static final Map<String, String> DEFAULT_PROPS = [
-    'netflix.environment'     : 'test',
-    'netflix.account'         : '${netflix.environment}',
-    'netflix.stack'           : 'test',
-    'spring.config.additional-location' : '${user.home}/.spinnaker/',
```
[the remark this answers is not in the document]
Comment (scottfrederick) [reply clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy]: This should be `additional-location`. halyard had `.node` but I don't think that's actually used. I'll make the change and test halyard again, and add `.node` just to halyard if it needs it.

**Comment by scottfrederick** [comment]
@ezimanyi see https://github.com/spinnaker/kork/pull/337

**Comment by ethanfrogers** [comment]
@ezimanyi we'll need to bump the `kork` version to account for spinnaker/kork#338. We should get this merged and then hit it with a follow up PR to bump to the next version.

**Comment by claymccoy** [comment]
Based on the previous discussion and the changes in kork this seems like it is ready to be merged. In fact, it seems like it may be holding back getting further improvements to kork.

**Comment by ethanfrogers** [comment]
Thanks @claymccoy!

**Comment by ethanfrogers** [comment]
@spinnakerbot cherry-pick 1.15

**Pull request merged**